devicetypes/plaidsystems/spruce-sensor.src/spruce-sensor.groovy

```diff
@@ -73,7 +73,7 @@ metadata {
 					[value: 64, color: "#44B621"],
 					[value: 80, color: "#3D79D9"],
 					[value: 96, color: "#0A50C2"]
-				]
+				], icon: 'http://www.plaidsystems.com/smartthings/st_spruce_leaf_225_top.png'
```

Comment:
There is a list of icons gathered by someone here: http://scripts.3dgo.net/smartthings/icons/
Someone else has also created: https://cdn.rawgit.com/JZ-SmartThings/SmartThings/master/Icons/ST-Icons.html

A good choice might be st.Outdoor.outdoor12 or st.Weather.weather12

That said, I think keeping custom icon assets available if used would fall under the WWST agreement requirement?

Anyway, if the Spruce logo is used, my recommendation is to make sure that it has the same aspect ratio and whitespace, as well as the same color, as the other icons.